Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.Tests/Lib/Helpers.cs

```diff
@@ -137,14 +138,20 @@ internal static void UploadSingleTestFile(string dir, string pathToFile)
         }
     }
 
-    internal static void CreateDummyFiles(int count)
+    internal static Array CreateDummyFiles(int count)
```

Comment:
You should return `string[]` instead of generic `Array`. If you return generic array then the class using the method doesn't know what type of values are stored in the array.